opm/simulators/utils/MPIPacker.hpp

```diff
@@ -59,9 +67,14 @@ struct Packing<true,T>
     //! \param comm The communicator to use
     static std::size_t packSize(const T*, std::size_t n, Parallel::MPIComm comm)
     {
+        // For now we do not handle the situation where a a single call to packSize/pack/unpack
+        // is likely to require an MPI_Pack_size value larger than intmax
+        assert ( n*sizeof(T) <= std::numeric_limits<int>::max() );
```

Comment:
Here we should probably throw an exception instead. Otherwise this will go undetected if we compile with `-DNDEBUG` (not default).

Reply:
Good point, will update.